### PR TITLE
Added PiVPN unattended installation option, Youtube video link added for Tailscale

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

File: docs/software/vpn.md [hunk 1]
```diff
@@ -114,9 +114,16 @@ PiVPN is an OpenVPN and WireGuard installer and management tool. It also has a c
 
     Simply run the command `pivpn -a`.
 
+=== "Unattended installation"
+
+    PiVPN offers the option of an non-interactive installation. For details read the [corresponding part of the PiVPN installation documentation](https://docs.pivpn.io/install/#non-interactive-installation).
```
Comment (MichaIng) [inline docs/software/vpn.md:119]:
```suggestion
    PiVPN offers the option of a non-interactive installation. For details read the [corresponding part of the PiVPN installation documentation](https://docs.pivpn.io/install/#non-interactive-installation).
```

File: docs/software/vpn.md [hunk 2]
```diff
@@ -114,9 +114,16 @@ PiVPN is an OpenVPN and WireGuard installer and management tool. It also has a c
...
+
+    This option can be achieved by placing a config file named `unattended_pivpn.conf` into the boot partition/directory. For example configs, have a look at https://github.com/pivpn/pivpn/tree/master/examples.
```
Comment (MichaIng) [inline docs/software/vpn.md:121]:
```suggestion
=== "Unattended installation"

    For an unattended PiVPN installation during first boot of DietPi, place a config file named `unattended_pivpn.conf` into the boot partition/directory. For example configs, have a look at <https://github.com/pivpn/pivpn/tree/master/examples>.

    More details can be found in the [corresponding part of the PiVPN installation documentation](https://docs.pivpn.io/install/#non-interactive-installation).
```

File: docs/software/vpn.md [hunk 3]
```diff
@@ -114,9 +114,16 @@ PiVPN is an OpenVPN and WireGuard installer and management tool. It also has a c
...
+
 ***
 
-Website: <https://www.pivpn.io>
+Website: <https://www.pivpn.io>
```
Comment (MichaIng) [inline docs/software/vpn.md:125]:
```suggestion
Website: <https://pivpn.io/>  
```

File: docs/software/vpn.md (hunk 2 above)
Comment (StephanStS) [reply docs/software/vpn.md:121]: Ok

**Comment by StephanStS** [comment]
@MichaIng: Please merge regardless of the link check problems.

**Review by MichaIng — APPROVED** [review]

**Pull request merged**